tests/unittests.c

```diff
@@ -67,11 +67,42 @@ static void cpu_freq_expect(const char *cpu_str, uint64_t expectation) {
     return;
 }
 
-static unsigned long test_kernel_task_func(void *arg) {
+static unsigned long test_kernel_task_func1(void *arg) {
     printk("CPU[%u]: Executing %s\n", smp_processor_id(), __func__);
     return _ul(arg);
 }
 
+static inline uint64_t rand47(void) {
+    return (_ul(rand()) << 16) ^ rand();
+}
+
+#define KT2_ROUNDS 100
+#define KT2_CHUNKS 10
+static unsigned long test_kernel_task_func2(void *arg) {
+    uint32_t seed;
+    asm("rdrand %0" : "=r"(seed));
+    srand(seed);
+
+    uint64_t blocks[KT2_CHUNKS] = {0};
+
+    for (size_t i = 0; i < KT2_ROUNDS; i++) {
+
```

Comment:
nit: extra newline

Reply:
Fixed in 6b1ebdc9cfacef472b23a47e141db6d56a4100d3